<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WalletMain</class>
 <widget class="QMainWindow" name="WalletMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WalletMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="saveOrLoadButton">
       <property name="text">
        <string>从文件加载</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generalButton">
       <property name="text">
        <string>新钱包</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mnemonicRecoveryButton">
       <property name="text">
        <string>从助记词恢复</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="status">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWallet_New">
   <property name="text">
    <string>Wallet New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
